### ISSUE TITLE
Address Case Details Critical Accessibility Issues

### ISSUE THREAD

**Issue by governmentSponsored** [US-Trustee-Program/Bankruptcy-Oversight-Support-Systems#912]
This ticket addresses accessibility issues that are WCAG violations. 

Overall
- [ ] Determine whether the sidebar navigation counts as repeated content across pages, for the sake of complying with WCAG 1.3.1 and 2.4.1.

Case Overview Issues
- [ ] Ensure that closing the staff assignment Edit modal returns focus back to the Edit button. (WCAG 2.4.3).
- [ ] Ensure that the focus indicator remains visible when tabbing backwards through the UI. Currently, it becomes hidden if the user shift+tabs up the list of tab stops, because it moves behind the heading overlay at the top of the screen. It only becomes visible again when focus moves above the viewport. (WCAG 2.4.11)

Court Docket issues
- [ ] Add an H3 above the filters that says "Filters". 
- [ ] Add helper text, below the Filters header, that says "As filters are applied, docket entries will be sorted or filtered out automatically." (WCAG 3.2.2).
- [ ] Update Find in Docket control to say "Find text in Docket". 
- [ ] Ensure each filter control has a basic accessible description that mentions that the controls will update the docket list automatically. (WCAG 3.2.2).
- [ ] Ensure the dropdown menus can be closed without shifting focus away from them (WCAG 2.4.3).
- [ ] Add an H3 above the first docket entry in the list, called "Docket Entries".
- [ ] Convert all docket entry numbers to H4 headings.
- [ ] Ensure the docket entry date and summary text is within an H5 heading (WCAG 1.3.1, 2.4.6).
- [ ] Ensure the Filter By Summary dropdown has the appropriate ARIA labels for custom widgets (WCAG 4.1.2)
- [ ] Ensure that fields that require specific input (e.g. Go To Document Number) programmatically inform the user that they do. (WCAG 3.3.3).
- [ ] Ensure text in button controls appropriate rescales when text spacing is adjusted (WCAG 1.4.12)
- [ ] Determine whether the list of docket entries should be considered an Ordered List, and thus use HTML list elements. (WCAG 1.3.1)
